### PR TITLE
Feature/ui size improvements

### PR DESCRIPTION
**Pull request by acepace**
# What is this? 
Improves on #633 by reducing size.
Develop = 3.4MiB
Lodash optimization = 3.35MiB
forteawesome optimization = 3.26MiB

### CODE REVIEW

**Comment by acepace** [comment]
Down to 2.39MiB
Not bad for 2 hours of work.

File: monkey/monkey_island/cc/ui/.babelrc [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
   "presets": [
-    "es2015",
-    "stage-0",
-    "react"
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry"
+      }
+    ],
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/.babelrc:8]: Can you tell me more about this change? preset-env by default parses into es2015 and what did stage-0 do ?

**Comment by acepace** [comment]
@ShayNehmad the build fails because I added react validation to eslint.

File: monkey/monkey_island/cc/ui/.babelrc (hunk 1 above)
Comment (acepace) [reply monkey/monkey_island/cc/ui/.babelrc:8]: stage-0 and es2015 were what babelrc transpiled into. All modern browsers already support it so there's nothing to transpire.
babel/preset-env lets babel target using browserslist (see change in package.json), which I defined to all browsers in use (0.25% and up). So not es2015.

Comment (acepace) [reply monkey/monkey_island/cc/ui/.babelrc:8]: Debug output shows that currently this means the following results

Using targets:
{
  "chrome": "49",
  "edge": "18",
  "firefox": "73",
  "ie": "11",
  "ios": "12",
  "opera": "66",
  "safari": "12.1",
  "samsung": "10.1"
}

**Comment by acepace** [comment]
@VakarisZ the two PTH icons are 1100x1100. Think it's ok to resize them or just to reformat them as 8bit colors rather than 32b, like the rest of the icons?

File: monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js [hunk 2]
```diff
@@ -346,7 +346,7 @@ class ConfigurePageComponent extends AuthComponent {
             throw Error()
           }
           return res;
-        }).catch(error => {
+        }).catch(() => {
         console.log('bad configuration');
         this.setState({lastAction: 'invalid_configuration'});
       }));
```
Comment (ShayNehmad) [inline monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js:352]: Log the error instead?

**Pull request merged**